instrumentation/instrumentation.gradle

```diff
@@ -53,6 +50,9 @@ shadowJar {
   mergeServiceFiles()
 
   exclude '**/module-info.class'
+  //These classes are added to bootstrap classloader by javaagent module
+  exclude 'io/opentelemetry/instrumentation/api/**/*.*'
+  exclude 'io/opentelemetry/javaagent/instrumentation/api/**/*.*'
```

Comment:
seems better (generally at least) to exclude at dependency level, can be done below:

```
  dependencies {
    exclude(project(':instrumentation-api'))
    exclude(project(':javaagent-api'))
    exclude(project(':javaagent-bootstrap'))
  }
```

Reply:
With the dependencies changed to `compileOnly` do we need any change here?

Reply:
it's also getting these dependencies through `:javaagent-tooling`